### DISCUSSION TITLE
No servers visible

### DISCUSSION THREAD

**Discussion by videobuff** [jamulussoftware/jamulus#1391] [Bug Investigation]
After succesfull installation on a mac I installed JAmulus on a windows machine. All went well, asio4al installed also, rebooted the machine, but when I launch Jamulus, I see no servers. However if i connect by simply hitting Enter, i am connected to a server, but I do not know which one.
Here is a link to this phenomena on YouTube: [https://youtu.be/RbP1P3L5Va0 ]

**Comment by gilgongo** [comment]
Have you ever been able to see a server list or has it just suddenly gone away? If you can't see any servers, you may want to look at the Troubleshooting page at the bottom: https://jamulus.io/wiki/Client-Troubleshooting : "Not seeing a list of servers at all?"

**Reply by videobuff** [reply]
No, not on this machine. SO i looked further and discovered that i could not ping my server, but i could see the webpage which is also running on the same machine. Checked DNS in the router and there it was ok, i could png direct from the router but not from the machine. Found the following commands and executed them as administrator frm a CMD window and the issue was solved.

pconfig /flushdns
ipconfig /registerdns
ipconfig /release
ipconfig /renew
NETSH winsock reset catalog
NETSH int ipv4 reset reset.log
NETSH int ipv6 reset reset.log
Exit

Thanks again
Erik

**Reply by gilgongo** [reply]
Interesting. Not sure I've seen anyone have local machine DNS lookup issues like that before, but I suppose it happens!
